dss/api/files.py

```diff
@@ -186,10 +186,30 @@ def put(uuid: str, extras: dict, version: str=None):
         FileMetadata.SHA256: metadata['hca-dss-sha256'],
     })
 
+    write_file_metadata(handle, dst_bucket, uuid, version, document)
```

Comment:
> Also, I'm slightly confused that you're not removing corresponding code from elsewhere. Where does the metadata write happen now?

the call is here, @kislyuk 

the diff here is not the best way to represent this code movement, unfortunately. :(